### PR TITLE
[DOCFIX] Update Chinese version of docs/en/Clients-S3.md 

### PR DESCRIPTION
**Pull request by minihippo**
Create docs/cn/Clients-S3.md corresponding to docs/en/Clients-S3.md

### CODE REVIEW

File: docs/cn/Clients-S3.md [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+
+<?xml version="1.0" encoding="UTF-8"?>
+<InitiateMultipartUploadResult xmlns="">
+  <Bucket>testbucket</Bucket>
+  <Key>testobject</Key>
+  <UploadId>2</UploadId>
+</InitiateMultipartUploadResult>
+```
+
+### Upload part
```
Comment (apc999) [inline docs/cn/Clients-S3.md]: can you translate this ?

File: docs/cn/Clients-S3.md [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+
+```bash
+# curl -i -X PUT http://localhost:39999/api/v1/s3/testbucket/testobject?partNumber=1&uploadId=2
+HTTP/1.1 200 OK
+Date: Tue, 29 Aug 2017 22:43:22 GMT
+ETag: "b54357faf0632cce46e942fa68356b38"
+Server: Jetty(9.2.z-SNAPSHOT)
+``` 
+
+### List parts
```
Comment (apc999) [inline docs/cn/Clients-S3.md]: translate

Comment (minihippo) [reply docs/cn/Clients-S3.md]: The file has been updated. Thanks for the comment.

File: docs/cn/Security.md [hunk 3]
```diff
@@ -106,6 +107,52 @@ Alluxio文件系统为目录和文件实现了一个访问权限模型，该模
...
+## 审查
+Alluxio支持审查日志用于系统管理员追踪用户对文件元数据的访问操作。
+
+审查日志文件(`master_audit.log`) 包括多个审查记录条目，每个条目对应一次文件元数据获取记录。
+Alluxio审查日志格式如下表所示：
+
+<table class="table table-striped">
+<tr><th>key</th><th>value</th></tr>
+<tr>
+  <td>succeeded</td>
```
Comment (yupeng9) [inline docs/cn/Security.md]: Can you also translate the content in this table?

Comment (minihippo) [reply docs/cn/Security.md]: The table has been translated. Thanks for the comment.

**Pull request merged**